src/containers/BooksForm.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import uuidv4 from 'uuid/v4';
+import PropTypes from 'prop-types';
+
+const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+class BookForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      category: '',
+    };
+    this.handleChangeInput = this.handleChangeInput.bind(this);
+    this.handleChangeSelect = this.handleChangeSelect.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChangeInput(event) {
+    this.setState({
+      title: event.target.value,
+    });
+  }
+
+  handleChangeSelect(event) {
+    this.setState({
+      category: event.target.value,
+    });
+  }
+
```

Comment:
You can condense these 2 in to ONE (see below) :
```
handleChange(e) {
    this.setState({
      [e.target.name]: e.target.value
    });
  };
```
and then bind the function to both `input` and `select` like this :
`onChange={this.handleChange.bind(this)}`

Reply:
Good idea! I like it!